src/server.c

```diff
@@ -3716,8 +3716,10 @@ void call(client *c, int flags) {
         }
     }
 
-    if (!(c->flags & CLIENT_BLOCKED))
+    if (!(c->flags & CLIENT_BLOCKED)) {
+        c->commands_processed++;
```

Comment:
I think we need to document that this does not include recursive commands. For example, when lua (or RM_Call) is executed recursively, lo `EVAL "server.call('ping')" 0` will increment this by 1, although "total commands" will get incremented by two.

Reply:
I thinks it's better for client and server to keep consistent with each other.

BTW, I've also been thinking that if introducing another metric to illustrate this issue would be better, such as `command_received` to represent commands received over the network, which can explicitly exclude the recursive commands.

Reply:
I'm fine either way. Just want to make sure we make it clear from documentation and in naming.

Reply:
Let's just keep this in the simple way and only add metrics that solves the problem directly.

Reply:
Ok. The consensus while meeting this morning was that we want cmds to be the same as the info command stats, so it should include recursive commands, since that will be clearest to the end user. Besides that, the name was fine.